string node1data1
string node1data2
